### ISSUE TITLE
Git update

### ISSUE THREAD

**Issue by zhuochun** [zhuochun/dotfiles#3]
```
[core]
        fsmonitor = true
[feature]
        manyFiles = true
```

```
brew link --force git

git config --global core.fsmonitor true
git config --global feature.manyFiles true
```

Explanation:

Certainly! Let's break down these `git config` commands:

1. `git config --global core.fsmonitor true`:
   - `git config`: This command is used to set or get configuration variables that control all aspects of how Git looks and operates.
   - `--global`: This flag specifies that the configuration should apply to the global configuration file, typically located at `~/.gitconfig`. This means the configuration will apply to all repositories for the current user unless overridden by repository-specific configurations.
   - `core.fsmonitor`: This is a configuration variable. When set to `true`, Git will use a file system monitor to watch for changes in the working directory. This can significantly speed up operations like `git status` in large repositories because Git can quickly determine which files have changed without scanning the entire working directory.
   
2. `git config --global feature.manyFiles true`:
   - `git config` and `--global`: As explained above.
   - `feature.manyFiles`: This is a configuration variable introduced in more recent versions of Git to optimize the performance for repositories with a large number of files. When set to `true`, it enables a set of default configuration options that are optimized for handling repositories with many files. This can include changes to index formats, optimizations for `git status`, and other performance-related tweaks.

In summary, both of these configurations are related to improving the performance of Git operations in repositories with a large number of files or large working directories.
